ksql-common/src/main/java/io/confluent/ksql/function/KsqlFunction.java

```diff
@@ -66,7 +102,8 @@ public static KsqlFunction createLegacyBuiltIn(
     };
 
     return create(
-        returnType, arguments, functionName, kudfClass, udfFactory, "", INTERNAL_PATH, false);
+        ignored -> returnType, arguments, functionName, kudfClass, udfFactory, "",
```

Comment:
nit: our general style is that if any arguments are on the next line, all arguments have their own line
```suggestion
        ignored -> returnType, 
        arguments, 
        functionName,
         kudfClass, 
         udfFactory, 
         "",
```